graphics/argyll/Portfile

```diff
@@ -80,11 +80,11 @@ post-destroot {
         ${destroot}${prefix}/share/${name}/ref
     # xinstall does not remove 'quarantine' attribute;
     # do that manually.
-    fs-traverse fixfile ${destroot} {
-        if {[exec xattr -l ${fixfile}] != ""} {
-            system "xattr -d com.apple.quarantine ${fixfile}"
-        }
-    }
+    #fs-traverse fixfile ${destroot} {
+    #    if {[exec xattr -l ${fixfile}] != ""} {
+    #        system "xattr -d com.apple.quarantine ${fixfile}"
+    #    }
+    #}
```

Comment:
it it's no longer needed it should be removed not commented out.

Reply:
at least that doesn’t work on my system - but I don’t have access to different system configurations so I can’t validate it might be necessary elsewhere…

Reply:
Well, presumably some version of this port's distfile contained files that had the com.apple.quarantine attribute set, and the portfile used `xattr` to remove them. You can check if the current distfile has any files that have that attribute, and if not, then it is safe to remove these lines (and please also remove the two comment lines above it).

If any files in the archive do still have com.apple.quarantine set, and those files get installed, then we still want to use `xattr` to remove them. We also want to see a bug report URL showing where the developers have been notified of the problem. And if using `xattr` "doesn't work on [your] system", show us the error message.